src/contracts/vote.ts

```diff
@@ -114,7 +113,7 @@ export class Vote implements UpdateableNetwork {
   public async get(proposalId: BigNumberish): Promise<Proposal> {
     const all = await this.getAll();
     const proposals = all.filter((p) =>
-      deepEqual(BigNumber.from(p.proposalId), BigNumber.from(proposalId)),
+      p.proposalId.eq(BigNumber.from(proposalId)),
```

Comment:
@joaquim-verges let me know if there was a specific reason we were using deepEqual to compare 2 BigNumbers? they come with their own `eq` method so this was confusing to me while I was cleaning up dependencies (all tests still pass)

Reply:
looks like an old leftover - def not needed here